inference-engine/tests/functional/plugin/cpu/single_layer_tests/eltwise.cpp

```diff
@@ -76,7 +76,9 @@ class EltwiseLayerCPUTest : public testing::WithParamInterface<EltwiseLayerCPUTe
         }
 
         std::shared_ptr<ngraph::Node> secondaryInput;
-        if (eltwiseType == ngraph::helpers::EltwiseTypes::DIVIDE ||
+        if (eltwiseType == ngraph::helpers::EltwiseTypes::ERF) {
```

Comment:
You shouldn't extend Eltwise test for ERF support. Right place would be: https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/plugin/cpu/single_layer_tests/activation.cpp

Reply:
done, thanks!